### PR TITLE
Jarv/provision

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: playbooks/library/ec2 [hunk 1]
```diff
@@ -0,0 +1,679 @@
+#!/usr/bin/python
+# This file is part of Ansible
```
Comment (feanil) [inline playbooks/library/ec2:2]: If this is a part of ansible why do we have to include it here?  Is it not available until a later version?

File: playbooks/roles/edxapp/defaults/main.yml [hunk 2]
```diff
@@ -293,77 +293,34 @@ sandbox_post_requirements:  "{{ edx_platform_code_dir }}/requirements/edx-sandbo
...
-  - vagrant
-  - yui-compressor
-  - zip
-  - zlib1g-dev
-
-# Ruby Specific Vars
+  # for shapely
+  - libgeos-dev
+
+# R by Specific Vars
```
Comment (feanil) [inline playbooks/roles/edxapp/defaults/main.yml]: missing 'u' in ruby.

File: playbooks/roles/gh_users/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+
+# gh_users
```
Comment (feanil) [inline playbooks/roles/gh_users/tasks/main.yml:3]: Can we remove this from the common role once this exists?

Comment (jarv) [inline playbooks/roles/gh_users/tasks/main.yml:3]: Yeah, I would prefer to not do it in this pr but we can later.

File: util/jenkins/ansible-provision.sh [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+EOF
+
+
+cat $extra_vars
+
+
+cd playbooks/edx-east
+# run the tasks to launch an ec2 instance from AMI
+ansible-playbook -vvvv edx_provision.yml  -i inventory.ini -e "@${extra_vars}"  --user ubuntu
+# run tasks to update application config files that
```
Comment (feanil) [inline util/jenkins/ansible-provision.sh]: rest of the sentence?

**Comment by feanil** [comment]
@jarv just a few nitpicks but otherwise looks good.  Please make an issue for pulling out the git_hub users content from the common role and then feel free to merge.

**Pull request merged**